staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -327,14 +334,19 @@ func patchResource(
 			if err := applyPatchToObject(codec, defaulter, currentObjMap, originalPatchMap, versionedObjToUpdate, versionedObj); err != nil {
```

Comment:
Now this line always fails for crds because the merge of https://github.com/kubernetes/kubernetes/pull/53558

Reply:
@nikhita